apps/worker/worker_core/config.py

```
@@ -1,24 +1,32 @@
+    CONF_DIR: str = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "conf")
 
-# File export config
-DOWNLOAD_DIR = env.get("DOWNLOAD_DIR", "/tmp")
-COOKIE_FILE_PATH = env.get("COOKIE_FILE_PATH", os.path.join(CONF_DIR, "cookies.txt"))
-PROXY_MODE = get_env_bool(env, "PROXY_MODE", False)
-PROXY_URL = env.get("PROXY_URL", "")
-YOUTUBE_COOKIE = get_env_bool(env, "YOUTUBE_COOKIE", False)
-BILIBILI_COOKIE = get_env_bool(env, "BILIBILI_COOKIE", False)
-OPENAI_API_KEY = env.get("OPENAI_API_KEY", "")
+    # File export
+    DOWNLOAD_DIR: str = "/tmp"
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)